KMB/kmb_massload.py

```diff
@@ -161,7 +161,7 @@ def process_date(entry):
 
 def process_byline(entry):
     """Handle unknown entries and rearrange names."""
-    if entry['byline'] in ('Okänd, Okänd', 'Okänd'):
+    if entry['byline'] in ('Okänd, Okänd', 'Okänd', 'Okänd Fotograf'):
```

Comment:
I'm pretty sure that the first two are actually the only that appear but I still like your solution.